network/endpoint_windows.go

```diff
@@ -64,16 +64,16 @@ func ConstructEndpointID(containerID string, netNsPath string, ifName string) (s
 }
 
 // newEndpointImpl creates a new endpoint in the network.
-func (nw *network) newEndpointImpl(cli apipaClient, _ netlink.NetlinkInterface, _ platform.ExecClient, _ netio.NetIOInterface, _ EndpointClient, epInfo *EndpointInfo) (*endpoint, error) {
-	if useHnsV2, err := UseHnsV2(epInfo.NetNsPath); useHnsV2 {
+func (nw *network) newEndpointImpl(cli apipaClient, _ netlink.NetlinkInterface, _ platform.ExecClient, _ netio.NetIOInterface, _ EndpointClient, epInfo []*EndpointInfo) (*endpoint, error) {
+	if useHnsV2, err := UseHnsV2(epInfo[0].NetNsPath); useHnsV2 {
```

Comment:
can you add comment here saying "assuming only single nic for windows..changes will be added later when doing for windows"